tensorbay/cli/cli.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. All Rights Reserved.
+#
+# pylint: disable=import-outside-toplevel
+
+"""Command-line interface.
+
+Use 'gas' + COMMAND in terminal to operate on datasets.
+
+Use 'gas config' to  configure environment.
+
+Use 'gas create' to create a dataset.
+
+Use 'gas delete' to delete a dataset.
+
+Use 'gas ls' to list data.
+
+"""
+
+import logging
+from typing import Dict
+
+import click
+
+from ..client import config as client_config
+from . import __version__
+
+
+@click.group()
+@click.version_option(__version__)
+@click.option("-k", "--key", "access_key", type=str, default="", help="The accessKey of gas.")
+@click.option("-u", "--url", type=str, default="", help="The login url.", hidden=True)
+@click.option(
+    "-p",
+    "--profile",
+    "profile_name",
+    type=str,
+    default="default",
+    help="The environment to login.",
+)
+@click.option("-d", "--debug", is_flag=True, help="Debug mode.")
+@click.pass_context
+def cli(ctx: click.Context, access_key: str, url: str, profile_name: str, debug: bool) -> None:
+    """You can use 'gas' + COMMAND to operate on your dataset.\f
+
+    Arguments:
+        ctx: The context to be passed as the first argument.
+        access_key: The accessKey of gas.
+        url: The login URL.
+        profile_name: The environment to login.
+        debug: Debug mode flag.
+
+    """  # noqa: D301,D415
+    ctx.obj = {
+        "access_key": access_key,
+        "url": url,
+        "profile_name": profile_name,
+    }
+    client_config._x_source = "PYTHON-CLI"  # pylint: disable=protected-access
+
+    if debug:
+        logging.basicConfig(level=logging.DEBUG)
```

Comment:
also move the implement of this function to other file, maybe move to `utility.py`